app-creator/src/widgets/search-bar/search-bar.ts

```diff
@@ -24,21 +24,23 @@ export class Searchbar extends LitElement {
       width: calc(100% - 2 * var(--tight));
       border: 0.7px solid rgba(0, 0, 0, 0.3);
       padding: var(--tight);
-      border-radius: var(--loose);
-      height: 20px;
+      border-radius: var(--regular);
```

Comment:
Could we rename it to `--regular-border-radius`?

Reply:
the regular variable is general for 16px and used in multiple places. I have created a new variable `--regular-border-radius` for this.